SU2_CFD/src/solvers/CTurbSASolver.cpp

```diff
@@ -803,7 +809,10 @@ void CTurbSASolver::BC_Engine_Inflow(CGeometry *geometry, CSolver **solver_conta
 
 //      /*--- Viscous contribution, commented out because serious convergence problems ---*/
 //
-//      visc_numerics->SetCoord(geometry->nodes->GetCoord(iPoint), geometry->nodes->GetCoord(iPoint));
```

Comment:
Here I also replaced the second `GetCoord(iPoint)` by `GetCoord(Point_Normal)` because it sound logical to me. I'm not sure whether this additional change is correct.

Reply:
When the distance is 0 the viscous numerics do not compute a Jacobian, I wonder if that has anything to do with the "serious convergence problems".
I also do not know if it is correct, but it is commented out code so... it does not matter I guess.